### PR TITLE
HDFS-16592.Fix typo for BalancingPolicy.

### PR DESCRIPTION
**Pull request by jianghuazhu**
### Description of PR
Fixed some typos related to BalancingPolicy.

### How was this patch tested?
Not too stressful for testing.

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime |  Logfile | Comment |
|:----:|----------:|--------:|:--------:|:-------:|
| +0 :ok: |  reexec  |   0m 51s |  |  Docker mode activated.  |
|||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  |  No case conflicting files found.  |
| +0 :ok: |  codespell  |   0m  1s |  |  codespell was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  |  The patch does not contain any @author tags.  |
| -1 :x: |  test4tests  |   0m  0s |  |  The patch doesn't appear to include any new or modified tests. Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch.  |
|||| _ trunk Compile Tests _ |
| +1 :green_heart: |  mvninstall  |  40m 30s |  |  trunk passed  |
| +1 :green_heart: |  compile  |   1m 27s |  |  trunk passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  compile  |   1m 17s |  |  trunk passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  checkstyle  |   1m  6s |  |  trunk passed  |
| +1 :green_heart: |  mvnsite  |   1m 25s |  |  trunk passed  |
| +1 :green_heart: |  javadoc  |   1m  8s |  |  trunk passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javadoc  |   1m 30s |  |  trunk passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  spotbugs  |   3m 21s |  |  trunk passed  |
| +1 :green_heart: |  shadedclient  |  21m 48s |  |  branch has no errors when building and testing our client artifacts.  |
|||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   1m 12s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 18s |  |  the patch passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javac  |   1m 18s |  |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 11s |  |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  javac  |   1m 11s |  |  the patch passed  |
| +1 :green_heart: |  blanks  |   0m  0s |  |  The patch has no blanks issues.  |
| +1 :green_heart: |  checkstyle  |   0m 51s |  |  the patch passed  |
| +1 :green_heart: |  mvnsite  |   1m 14s |  |  the patch passed  |
| +1 :green_heart: |  javadoc  |   0m 50s |  |  the patch passed with JDK Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1  |
| +1 :green_heart: |  javadoc  |   1m 22s |  |  the patch passed with JDK Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07  |
| +1 :green_heart: |  spotbugs  |   3m 10s |  |  the patch passed  |
| +1 :green_heart: |  shadedclient  |  21m 42s |  |  patch has no errors when building and testing our client artifacts.  |
|||| _ Other Tests _ |
| -1 :x: |  unit  | 255m 28s | [/patch-unit-hadoop-hdfs-project_hadoop-hdfs.txt](https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4351/1/artifact/out/patch-unit-hadoop-hdfs-project_hadoop-hdfs.txt) |  hadoop-hdfs in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 55s |  |  The patch does not generate ASF License warnings.  |
|  |   | 361m 33s |  |  |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdfs.TestClientProtocolForPipelineRecovery |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4351/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/4351 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient spotbugs checkstyle codespell |
| uname | Linux 9fc2a6dd2321 4.15.0-169-generic #177-Ubuntu SMP Thu Feb 3 10:50:38 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/bin/hadoop.sh |
| git revision | trunk / 8081ef3ae322798ebaf16b7d054212ce50077d8a |
| Default Java | Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
| Multi-JDK versions | /usr/lib/jvm/java-11-openjdk-amd64:Private Build-11.0.15+10-Ubuntu-0ubuntu0.20.04.1 /usr/lib/jvm/java-8-openjdk-amd64:Private Build-1.8.0_312-8u312-b07-0ubuntu1~20.04-b07 |
|  Test Results | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4351/1/testReport/ |
| Max. process+thread count | 4088 (vs. ulimit of 5500) |
| modules | C: hadoop-hdfs-project/hadoop-hdfs U: hadoop-hdfs-project/hadoop-hdfs |
| Console output | https://ci-hadoop.apache.org/job/hadoop-multibranch/job/PR-4351/1/console |
| versions | git=2.25.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.14.0-SNAPSHOT https://yetus.apache.org |


This message was automatically generated.

**Comment by jianghuazhu** [comment]
Here are some failing unit tests such as:
TestWebHDFS
TestUnderReplicatedBlocks
TestExternalStoragePolicySatisfier
TestIncrementalBlockReports
TestRedudantBlocks

It looks like these failures have little to do with the code I submitted.
@aajisaka  @ferhui , can you help review this pr?
Thank you very much.

File: hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/balancer/BalancingPolicy.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
  * Balancing policy.
  * Since a datanode may contain multiple block pools,
  * {@link Pool} implies {@link Node}
- * but NOT the other way around
+ * but not the other way around
```
Comment (ayushtkn) [inline hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/balancer/BalancingPolicy.java:31]: I think this is just changing the case of Not, this much we can tolerate and feels like it was intentionally done to increase the weight of not or bring into notice. this isn't a typo nor a grammatical error by my books, just a way of writing which in general world is acceptable.

Comment (jianghuazhu) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/balancer/BalancingPolicy.java:31]: OK
I think I learned something new.
Thanks @asymc for the guidance.
I will update the status of jira later.

Comment (jianghuazhu) [reply hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/balancer/BalancingPolicy.java:31]: I think I learned something new.
Thanks @ayushtkn  for the guidance.
I will update the status of jira later.

**Pull request closed without merging**